libs/build_dod_dataset.py

```diff
@@ -49,145 +52,82 @@ def _get_abbrev_df():
-
-    # USA only
-    us_df = remapped_df[(remapped_df["Country/Region"] == "US")]
-    jhu_column_names = [
-        "Province/State",
-        "Country/Region",
-        "Last Update",
-        "Latitude",
-        "Longitude",
-        "Confirmed",
-        "Recovered",
-        "Deaths",
-        "Active",
-        "County",
-        "State/County FIPS Code",
-        "Combined Key",
-        # Incident rate and people tested do not seem to be available yet
-        # "Incident Rate",
-        # "People Tested",
-    ]
-    final_df = pd.DataFrame(us_df, columns=jhu_column_names)
-    final_df["Last Update"] = pd.to_datetime(final_df["Last Update"])
-    final_df["Last Update"] = final_df["Last Update"].dt.strftime("%-m/%-d/%Y %H:%M")
-
-    final_df["County"] = final_df["County"].replace(county_replace_with_null)
-    final_df["Combined Key"] = final_df["Combined Key"].str.replace("Unassigned, ", "")
-    final_df = final_df.fillna(NULL_VALUE)
-    final_df = final_df.drop_duplicates(
-        "State/County FIPS Code"
-    )  # note this is a hack, 49053 is dupped in JHU data :(
-    final_df.index.name = "OBJECTID"
-    # assert unique key test
-    assert final_df["Combined Key"].value_counts().max() == 1
-    assert final_df["State/County FIPS Code"].value_counts().max() == 1
+    data = JHUDataset.local().timeseries()
+    data = data.get_subset(None, country="USA")
+    return data.latest_values(AggregationLevel.COUNTY)
+
 
-    return final_df
+def _get_usa_by_states_df():
+    data = JHUDataset.local().timeseries()
+    data = data.get_subset(None, country="USA")
+    return data.latest_values(AggregationLevel.STATE)
 
 
 def get_usa_by_county_with_projection_df(input_dir, intervention_type):
     us_only = _get_usa_by_county_df()
-    fips_df = FIPSPopulation.local().data  # used to get interventions
-    interventions_df = (
-        _get_interventions_df()
-    )  # used to say what state has what interventions
+    fips_df = FIPSPopulation.local().data
+    # used to get interventions
+    interventions_df = _get_interventions_df()
     projections_df = get_county_projections_df(
         input_dir, intervention_type, interventions_df
     )
 
     counties_decorated = (
         us_only.merge(
             projections_df,
-            left_on="State/County FIPS Code",
+            left_on=TimeseriesDataset.Fields.FIPS,
             right_on="FIPS",
             how="inner",
         )
-        .merge(fips_df[["state", "fips"]], left_on="FIPS", right_on="fips", how="inner")
         .merge(interventions_df, left_on="state", right_on="state", how="inner")
     )
-
     counties_remapped = counties_decorated.rename(
         columns=OUTPUT_COLUMN_REMAP_TO_RESULT_DATA
     )
     counties = pd.DataFrame(counties_remapped, columns=RESULT_DATA_COLUMNS_COUNTIES)
     counties = counties.fillna(NULL_VALUE)
     counties.index.name = "OBJECTID"
+    counties["Province/State"] = counties["Province/State"].map(abbrev_us_state)
```

Comment:
i mean would love to use the class vars here if we have them?

Reply:
word. I think I'll be able to use your latest PR for that right?